pre-commit-8-4

```diff
@@ -14,7 +14,7 @@ command_exists () {
 # Function to search array.
 contains_element () {
   local e
-  for e in "${@:2}"; do [[ "$e" == "$1" ]] && return 0; done
+  for e in ${@:2}; do [[ "$e" == "$1" ]] && return 0; done
```

Comment:
When both `"${FILES[@]}"` and `"${@:2}"` are double quoted, the list of files is not split correctly and this function always returned 1.

Reply:
Tested this bit locally by doing
```
#!/bin/bash
# Function to search array.
contains_element () {
  local e
  for e in ${@:2}; do [[ "$e" == "$1" ]] && return 0; done
  return 1
}
FILES=$(ls);

contains_element "not_there" "${FILES[@]}"
echo $?

contains_element "there" "${FILES[@]}"
echo $?
```
And confirm that this change at least makes contains_element() work as expected. Nice.

It didn't really matter before it meant that we were doing more compile checks than necessary and potentially told a committer twice about the same problem.